src/viam/sdk/tests/test_servo.cpp

```diff
@@ -95,6 +97,7 @@ BOOST_AUTO_TEST_SUITE(test_servo_client_server)
 template <typename Lambda>
 void server_to_mock_pipeline(Lambda&& func) {
     ServoServer servo_server;
+    std::shared_ptr<MockServo> mock = MockServo::get_mock_servo();
```

Comment:
(nit) doesn't look like this `mock` is used anywhere, I think we can get rid of it.